projects/1.18/assets/ars-nouveau/ars_nouveau/lang/zh_cn.json

```
@@ -769,62 +770,62 @@
   "entity.ars_nouveau.flourishing_weald_walker": "繁茂林地行者",
   "entity.ars_nouveau.blazing_weald_walker": "烈焰林地行者",
   "entity.ars_nouveau.vexing_weald_walker": "恼人林地行者",
-  "ars_nouveau.page1.weald_walker": "林地行者是一种可活动的至高木，其行为主要是从敌对怪物中保护自己的家园。可以用唤醒仪式创造林地行者。被召唤出来后，它们会无目的地随机漫步，除非有人给它们指定家的位置。要给林地行者指定家的位置，需要先对林地行者，再对你想要保护的方块使用支配之杖。",
-  "ars_nouveau.page2.weald_walker": "能对附近敌人施展爆燃，而且会小心不点燃任何方块。",
+  "ars_nouveau.page1.weald_walker": "林地行者是一种可活动的至高木树人，能从敌对怪物中保护自己的家园。可以用唤醒仪式创造林地行者。刚召唤出的林地行者只会漫无目的的游荡，使用支配之杖给为它们指定坐标之后，它们就会开始守卫这个地方。",
+  "ars_nouveau.page2.weald_walker": "能对附近敌人施展爆燃，而且不会点燃任何方块。",
   "ars_nouveau.page3.weald_walker": "能对附近敌人施展冻结和寒流。",
   "ars_nouveau.page4.weald_walker": "能对附近敌人施展强化过的伤害和禁锢。",
   "ars_nouveau.page5.weald_walker": "能对附近敌人施展脆弱诅咒和强化过的凋零。",
-  "ars_nouveau.page1.weald_waddler": "如果林地行者死亡，它会变成林地蹒跚者。林地蹒跚者又会慢慢长大，变回林地行者，对其使用骨粉可以加速这个过程。林地蹒跚者在长大前无法战斗，也不能自保。",
+  "ars_nouveau.page1.weald_waddler": "如果林地行者死亡，它会变成林地蹒跚者，林地蹒跚者会缓慢生长变回林地行者。可以使用骨粉加速林地蹒跚者的恢复。林地蹒跚者本身不能战斗，也没有自保能力。",
   "ars_nouveau.source": "魔源消耗：%s",
   "ars_nouveau.crush_recipe": "魔符：粉碎",
   "ars_nouveau.enchanting_apparatus": "附魔装置",
   "ars_nouveau.page.apparatus_crafting": "附魔装置合成",
-  "ars_nouveau.page1.apparatus_crafting": "附魔装置的用途是合成新生魔艺中使用的特殊机器、饰品和装备。使用其进行合成需要最多8个奥术基座、一个奥术核心，还有附魔装置方块本身。放置好附魔装置运行需要的方块后，你应该先合成你的第一个法师之花种子。",
+  "ars_nouveau.page1.apparatus_crafting": "附魔装置可以合成新生魔艺中的特殊机器、饰品和装备。需要最多8个奥术基座、一个奥术核心，还有附魔装置方块本身。你需要制作的第一个配方是法师之花种子。",
   "ars_nouveau.page.better_casting": "更好地施法",
-  "ars_nouveau.page1.better_casting": "你的魔力池可以通过特殊法师盔甲、附魔、学习新魔符、饮用药水等方式拓展。获得法师之花种子后，你可以合成初学者法袍，这套装备会显著提升你的施法能力。这些法袍不但有着较高的附魔性能和护甲值，还能使用你的魔力进行自我修复。",
+  "ars_nouveau.page1.better_casting": "你的魔力可以通过特殊法师盔甲、附魔、学习新魔符、饮用药水等方式拓展。获得法师之花种子后，你可以合成法袍，这套装备会显著提升你的施法能力。这些法袍不但有着较高的附魔性能和护甲值，还能使用你的魔力进行自我修复。",

```

此处为模组手册内容，类似新手教程，建议遵照原文
从这里到L882同理，请自查
```suggestion
  "ars_nouveau.page1.better_casting": "你的魔力可以通过特殊法师盔甲、附魔、学习新魔符、饮用药水等方式拓展。获得法师之花种子后，你可以合成学徒法袍，这套装备会显著提升你的施法能力。这些法袍不但有着较高的附魔性能和护甲值，还能使用你的魔力进行自我修复。",
```